src/main/java/org/terasology/minimap/rendering/nui/layers/MinimapGrid.java

```diff
@@ -84,6 +83,9 @@
     private Binding<EntityRef> targetEntityBinding = new DefaultBinding<>(EntityRef.NULL);
     private Binding<Integer> zoomFactorBinding = new DefaultBinding<>(0);
 
+    private Set<EntityRef> alivePlayers = new HashSet<EntityRef>();
```

Comment:
Can be 
```suggestion
    private final Set<EntityRef> alivePlayers = new HashSet<EntityRef>();
```

Reply:
If we keep it final, can it be changed? More players will be added as the join the game.

Reply:
I got confused between final and immutable. I will change it to final.